packages/web/src/components/treeView.css

```diff
@@ -69,6 +69,13 @@
   color: var(--vscode-list-activeSelectionForeground) !important;
 }
 
+.tree-view-content:focus .tree-view-entry.selected button.eye.toggled {
+  color: var(--vscode-button-secondaryForeground) !important;
```

Comment:
This color doesn't actually change anything other than add a semantic color name (the color stays the same). I'd rather avoid another `!important`.